<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-1</y>
      <width>601</width>
      <height>361</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page_1">
     <widget class="QPushButton" name="btTempDown">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>230</y>
        <width>80</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>DOWN</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btTempUp">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>230</y>
        <width>80</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>UP</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_Temp">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>230</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Temperature</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btCheckIn">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>190</y>
        <width>80</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>CHECK IN</string>
      </property>
     </widget>
     <widget class="QLabel" name="DispTemp">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>40</y>
        <width>101</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #445566;</string>
      </property>
      <property name="text">
       <string>0</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="DispSettemp">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>40</y>
        <width>101</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #665544;</string>
      </property>
      <property name="text">
       <string>0</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="DispWdspd">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>40</y>
        <width>101</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #556644;</string>
      </property>
      <property name="text">
       <string>0</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="DispSetwdspd">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>40</y>
        <width>101</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #446655;</string>
      </property>
      <property name="text">
       <string>0</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
